### PR TITLE
feat(validateType): add build/ci validate type 

### PR DESCRIPTION
**Pull request by jackyr**
Add build/ci validate type #47

### CODE REVIEW

**Comment by kentcdodds** [comment]
Hi @jackyr,
Actually, that's not how you would update this tool to support `build/ci`. Let me comment in line.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ You can specify options in `package.json`
 {
   "config": {
     "validate-commit-msg": {
-      "types": ["feat", "fix", "docs", "style", "refactor", "perf", "test", "chore", "revert"], // default
+      "types": ["feat", "fix", "docs", "style", "refactor", "perf", "test", "build", "ci", "chore", "revert"], // default
```
Comment (kentcdodds) [inline README.md]: This is just documentation for how you might configure things in your own project to add different types. It's not required, but if you wanted to support these types in your project, you could add this to your package.json.

File: package.json [hunk 2]
```diff
@@ -60,6 +60,8 @@
         "refactor",
         "perf",
         "test",
+        "build",
+        "ci",
```
Comment (kentcdodds) [inline package.json]: This is the configuration for `validate-commit-msg` which uses itself to validate commit messages. This wont have any impact on consumers of the library.

**Comment by jackyr** [comment]
I see. Thank you!

**Pull request closed without merging**